var/spack/repos/builtin/packages/kaldi/0002-configure-Fix-gcc-version-check-for-cuda.patch

```diff
@@ -0,0 +1,39 @@
+From b3b9c5c7cade0f964336ec5684cc5a7bc6abb10f Mon Sep 17 00:00:00 2001
+From: Roland Fehrenbacher <rf@q-leap.de>
+Date: Mon, 29 Jan 2024 15:01:33 +0100
+Subject: [PATCH 2/2] configure: Fix gcc version check for cuda
+
+---
+ src/configure | 7 ++++---
+ 1 file changed, 4 insertions(+), 3 deletions(-)
+
+diff --git a/src/configure b/src/configure
+index 37a75a5ca..82e4e3d91 100755
+--- a/src/configure
++++ b/src/configure
+@@ -389,8 +389,8 @@ Either your CUDA is too new or too old."
+           CUSOLVER=true
+         ;;
+         12_*)
+-          MIN_UNSUPPORTED_GCC_VER="12.2"
+-          MIN_UNSUPPORTED_GCC_VER_NUM=122000
++          MIN_UNSUPPORTED_GCC_VER="12.3"
++          MIN_UNSUPPORTED_GCC_VER_NUM=123000
+           CUSOLVER=true
+         ;;
```

Comment:
This *seems* like something that should be upstreamed.

Reply:
https://github.com/kaldi-asr/kaldi/pull/4905

Reply:
Wow, this was fast. The above PR was already merged. So please close this PR unmerged. I will submit a new one without this patch shortly.